lib/node.js

```diff
@@ -1654,6 +1654,68 @@ type net$connectOptions = {
   ...
 };
 
+declare module "module" {
+  declare type FilePath = string;
```

Comment:
I don't believe this is done anywhere else in this definition, e.g. the fs typings. Maybe just `string`?